app/views/deep_links/mobile.html.erb

```diff
@@ -0,0 +1,42 @@
+<div id="mobile-deep-link-spinner" class="flex flex-col">
+  <svg class="crayons-icon crayons-spinner" width="24" height="24" viewBox="0 0 24 24" aria-hidden="true" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M18.364 5.636L16.95 7.05A7 7 0 1019 12h2a9 9 0 11-2.636-6.364z" fill="currentColor" class="grow-1"></path></svg>
+  <p class="align-center">Attempting to open the mobile app...</p>
```

Comment:
```suggestion
  <p class="align-center">Opening the mobile app...</p>
```

Just a copy suggestion; "attempting to" makes it sound a little like it's likely to fail LOL 😂 🙊